toolchain/sem_ir/typed_insts.h

```diff
@@ -831,6 +831,13 @@ struct ImportDecl {
   NameId package_id;
 };
 
+// An `import Cpp` declaration.
+struct ImportCppDecl {
+  static constexpr auto Kind =
+      InstKind::ImportCppDecl.Define<Parse::ImportDeclId>(
+          {.ir_name = "import_cpp", .is_lowered = false});
+};
+
```

Comment:
Minor, I see you sorted elsewhere, but `ImportCppDecl` before `ImportDecl` here too

Reply:
Done.